iquip/apps/dataviewer.py

```diff
@@ -296,19 +297,28 @@ def __init__(self, parent: Optional[QWidget] = None):
         layout.addWidget(self.label)
         # signal connection
         self.button.toggled.connect(self._buttonToggled)
+        self.button.clicked.connect(functools.partial(self.button.setEnabled, False))
```

Comment:
Although I prefer `partial()` to `lambda`, in this case `lambda` might be a better solution since it strictly and explicitly show that it takes no argument, and `clicked` signal has 1 boolean argument.
`partial()` makes it obscure that if the boolean argument is consumed or not (of course we know that `self.button.setEnabled()` takes a single argument and it is given by `False` already, so the partial function will not take any more arguments).
However, I want to use `partial()` when I can ;)